index.bs

```diff
@@ -696,6 +698,39 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Signal platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method is used to signal users whether they can log in to the [RP] with the <a>platform-attached</a> authenticator. 
+The method gives [RP] the ability to detect whether the client is capable of registering credentials with 
+<a>platform authenticators</a> while giving users control over whether they want to share this information. When the 
+method is invoked, the <a>client</a> will search for the existence of an <a>platform authenticator</a> capable of 
+registering credentials. Upon finding one, the <a>client</a> will prompt the user for whether the user is interested
```

Comment:
Good point. I am thinking maybe reject the promise with a "NotAllowedError", which would be the default error for all situations? This way the RP won't know exactly whether it is the lack of authenticator that triggers the failure, thus preventing fingerprinting attack.